src/HomePage/HomePage.js

```diff
@@ -2,18 +2,29 @@ import React, { Component } from "react";
 import Header from "../Header/Header";
 import Footer from "../Footer/Footer";
 import MovieCard from "../MovieCard/MovieCard";
-import movieData from "../data/movies.js";
 import "./HomePage.css";
+import { getMovieData } from "../apiCalls";
 
 
 class HomePage extends Component {
   constructor(props) {
     super(props)
     this.state= {
-      movies: movieData.movies,
+      movies: [],
+      error: ''
```

Comment:
We've got some all movies living in HomePage, but the single movies live in App - what was your reason for this decision?

Reply:
Originally we had wanted the single movies state to live in the MovieSnapShot. We have sense moved its state there. We had an issue with page build timing that we were able to fix when refactoring to implement  the router.